doc/whats_new/v0.23.rst

```diff
@@ -317,6 +317,10 @@ Changelog
 - |API| Adds :class:`feature_selection.SelectorMixin` back to public API.
   :pr:`16132` by :user:`trimeta`.
 
+- |Feature| :class:`feature_selection.RFE`, :class:`feature_selection.RFECV` and
```

Comment:
We will need to move this in 0.24.
Could you limit to 79 characters for this and the next line